### ISSUE TITLE
Challenge Mode Only

### ISSUE THREAD

**Issue by Viskus** [ZhengPeiRu21/mod-challenge-modes#23]
Thanks for your work on this ZhengPieRu21
Hoping you might be able to implement a setting where when a hardcore mode is selected (potentially just one from the config) it then defaults to every character so that the server is challenge mode centric.
Or it could potentially not allow any XP to be given until a challenge mode is opted into?
Thanks again!

**Comment by Zindokar** [comment]
Yes, I wish they could implement hardcore by default, just on login, including an option with level but the opposite system they have, like activate the "hardcore" at lvl 80 (or customizable).
